jacodb-analysis/src/main/kotlin/org/jacodb/analysis/codegen/Generator.kt

```diff
@@ -196,21 +196,20 @@ private fun runCmd(
     val outputFile = File(outputFileName)
     try {
         val cmdBuilder = ProcessBuilder()
-        cmdBuilder.directory(workingDir)
-        cmdBuilder.command(cmd)
+        cmdBuilder.directory(workingDir).command(cmd)
         val cmdProcess = cmdBuilder.redirectError(errorFile).redirectOutput(outputFile).start()
         cmdProcess.waitFor()
         val hasErrors = errorFile.length() != 0L
         if (hasErrors) {
-            logger.error { "$errorMessage. check logs in: ${errorFile.path}" }
+            logger.error { "$errorMessage. check logs in - ${errorFile.path}" }
```

Comment:
1. rename `errorMessage` to `errorPrefix`
2. after `errorPrefix` use colon instead of dot